src/meta/meta_split_service.h

```diff
@@ -48,6 +57,19 @@ class meta_split_service
     void
     on_add_child_on_remote_storage_reply(error_code ec, register_child_rpc rpc, bool create_new);
 
+    static const std::string control_type_str(split_control_type::type type)
```

Comment:
What about use `ENUM_REG` for `split_control_type`?

Reply:
If `split_control_type` is defined by ENUM_REG, the string will be like `dsn::replication::split_control_type::PSC_PAUSE`, the function `control_type_str` is only used when printing log or hint message to show the control type, I think the string using ENUM_REG is a little bit too long.